plugins/fields/radio/radio.php

```diff
@@ -41,9 +41,21 @@ public function onCustomFieldsBeforePrepareField($context, $item, $field)
         if (!$this->isTypeSupported($field->type)) {
             return;
         }
+        
+        $field->apivalue = [];
```

Comment:
```suggestion

        $field->apivalue = [];
```
Remove white space from empty line to fix code style error reported by drone.